src/mono/mono.proj

```diff
@@ -87,6 +87,40 @@
                       GeneratePathProperty="true" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(TargetsBrowser)' == 'true'">
+    <NodePkgVersion Condition="'$(HostOS)' == 'linux' and '$(BuildArchitecture)' == 'x64'">$(runtimelinuxx64MicrosoftNETCoreRuntimeWasmNodeTransportVersion)</NodePkgVersion>
```

Comment:
https://learn.microsoft.com/en-us/visualstudio/msbuild/choose-element-msbuild?view=vs-2022 might make this easier to express and less typo prone

Reply:
updated. it is a bit verbose though